private/aws-client-retry-test/src/ClientRetryTest.spec.ts

```diff
@@ -89,21 +89,22 @@ describe("util-retry integration tests", () => {
     } catch (error) {
       expect(error).toStrictEqual(expectedException);
       expect(error.$metadata.httpStatusCode).toBe(429);
-      expect(error.$metadata.attempts).toBe(4);
+      expect(error.$metadata.attempts).toBe(3);
       expect(error.$metadata.totalRetryDelay).toBeGreaterThan(0);
     }
   });
 
   it("should use a shared capacity for retries", async () => {
     const expectedInitialCapacity = 500;
     const expectedDrainPerAttempt = 5;
-    const expectedRetryAttemptsPerRequest = 7;
+    const expectedRetryAttemptsPerRequest = 6;
```

Comment:
nit: create another constant for `maxAttempts` instead of using `expectedRetryAttemptsPerRequest` to calculate it.

Reply:
fixed in 397b004bbb699eb51281598f3c155b61a9eff3ce.

Separating out maxAttempts is much more clear.